tools/monitoring/gke_deployment/monitoring_service.yaml

```diff
@@ -41,11 +41,44 @@ spec:
       containers:
       - name: monitoring-app
         image: gcr.io/knative-tests/test-infra/monitoring:latest
+        env:
+        - name: MONITORING_DB_NAME
+          value: monitoring
+        - name: MONITORING_DB_HOST
+          value: "127.0.0.1"
+        - name: MONITORING_DB_PORT
+          value: "3306"
+        - name: MONITORING_DB_USER
+          valueFrom:
+            secretKeyRef:
+              name: monitoring-db-credentials
+              key: username
+        - name: MONITORING_DB_PASSWORD
```

Comment:
We should access this as a secret and not an env var

Reply:
Done